### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by NathanOrawi**
Please review my code @NathanTestHub

### CODE REVIEW

File: prac_04/quick_picks.py [hunk 1]
```diff
@@ -1,16 +1,11 @@
 from random import randint
 
 # Write a program that asks the user how many "quick picks" they wish to generate
-QUICK_PICKS = []
+quick_picks = []
```
Comment (NathanTestHub) [inline prac_04/quick_picks.py]: why two quick_pics

Comment (NathanTestHub) [reply prac_04/quick_picks.py]: fixed 👍

File: prac_04/quick_picks.py [hunk 2]
```diff
@@ -1,16 +1,11 @@
...
-
-# for quick_pick in range(6):
 for number_of_quick_pick in range(number_of_quick_picks):
     for number in range(6):
-        QUICK_PICK = [randint(1, 45)]
-        QUICK_PICKS += QUICK_PICK
-    quick_picks_row = (QUICK_PICKS[-6:])
-    print(sorted(quick_picks_row))
-    # for i in range(len(quick_picks_row)):
-    #     print(quick_picks_row[i])
```
Comment (NathanTestHub) [inline prac_04/quick_picks.py:16]: formatting will make code easier to read

**Review by NathanTestHub — COMMENTED** [review]
Nice job

**Review by NathanTestHub — COMMENTED** [review]
clean up code

File: prac_04/quick_picks.py (hunk 1 above)
Comment (NathanOrawi) [reply prac_04/quick_picks.py]: Thank you. What a cool guy 💯

**Pull request merged**